src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthModule.cs

```diff
@@ -297,7 +297,20 @@ public ResultWrapper<Keccak> eth_sendRawTransaction(byte[] transaction)
         public ResultWrapper<byte[]> eth_call(TransactionForRpc transactionCall, BlockParameter blockParameter = null)
         {
             BlockHeader block = blockParameter == null ? _blockchainBridge.Head : _blockchainBridge.GetBlock(blockParameter).Header;
-            BlockchainBridge.CallOutput result = _blockchainBridge.Call(block, transactionCall.ToTransaction());
+
+            var tx = transactionCall.ToTransaction();
```

Comment:
ToTransaction currently has: tx.GasLimit = (long)(Gas ?? 90000);

So if no Gaslimit was set it will be 90k, maybe change there?